pmrl.py

```diff
@@ -87,9 +87,13 @@ def draw_map(
             elif x == exit_coords.x and y == exit_coords.y:
                 tile = '<'
                 color = libtcod.green
-            libtcod.console_set_default_foreground(0, color)
-            libtcod.console_put_char(0, x, y, tile, libtcod.BKGND_NONE)
```

Comment:
All references to setting fg color and putting a char are replaced by `root_console.draw_rect()` with width=1/height=1. A character code is required by this function, for which we can simply use [builtin ord()](https://docs.python.org/3/library/functions.html#ord)